test/run.sh

```diff
@@ -0,0 +1,8 @@
+cd ..
```

Comment:
As far as I can tell the script is run from the root anyway. Please check the travis CI results.